GliaWidgets/Component/Header/Header.swift

```diff
@@ -50,23 +57,31 @@ class Header: UIView {
         updateHeight()
     }
 
-    func setLeftItem(_ item: UIView?, animated: Bool) {
-        setItem(item, to: leftItemContainer, animated: animated)
+    public func setLeftItem(animated: Bool) {
+        setItem(backButton, to: leftItemContainer, animated: animated)
     }
 
-    func setRightItem(_ item: UIView?, animated: Bool) {
-        setItem(item, to: rightItemContainer, animated: animated)
+    public func showCloseButton() {
+        endButton.isHidden = true
+        endScreenSharingButton.isHidden = true
+        closeButton.isHidden = false
     }
 
-    func setRightItems(_ items: [UIView]?, animated: Bool) {
-        var stack: UIStackView?
-        if let items = items {
-            stack = UIStackView(arrangedSubviews: items)
-            stack?.axis = .horizontal
-            stack?.spacing = 8
-            stack?.alignment = .center
-        }
-        setRightItem(stack, animated: animated)
+    func setEndButtons() {
```

Comment:
All your other functions that operate on the `isHidden` attribute are called `show***Button`, why is this one different then? :)
Obviously, something like `showEndEngagementAndEndScreenShareButtons` is way too long, but we can make it `showBothEndButtons` or something like this. I honestly don't like the "both" variant very much, but I wasn't able to produce anything more better-looking 😄

Reply:
I found out that I don't use this function at all😄